src/components/Discipline.vue

```diff
@@ -57,6 +57,16 @@ export default {
         )
         .map(({ id }) => id);
     },
+    computeBonus() {
+      /**
+       * compute bonus for predator discipline
+       * @returns {Int}
```

Comment:
neviem ci to VS code alebo codium bere Int ako platnu navratovu moznost, ked tak treba nad miestom kde to je pouzite chvilu hoverovat kym sa neukaze napoveda
pouzil by som bud `integer` alebo `Number`